src/OrchardCore/OrchardCore.DisplayManagement/Handlers/DisplayDriver.cs

```diff
@@ -115,12 +115,17 @@ public virtual Task<IDisplayResult> UpdateAsync(TModel model, IUpdateModel updat
 
         protected virtual void BuildPrefix(TModel model, string htmlFieldPrefix)
         {
-            Prefix = typeof(TModel).Name;
+            var builder = new StringBuilder();
```

Comment:
I would have to benchmark it to be sure, but I think you're probably increasing allocations by making this change to a stringbuilder.

Reply:
@deanmarcussen yes you were right. I thought because StringBuilder use Span behind the seance, I thought it would reduce allocation. But I think in this case it's not a huge string and only a single concatenation.

I remove StringBuilder and used concatenation instead. Let me know if you see anything else or if you approve.